source/reference/minio-mc-admin/mc-admin-scanner-trace.rst

```diff
@@ -0,0 +1,219 @@
+==========================
+``mc admin scanner trace``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin scanner trace
+
+Description
+-----------
+
+.. start-mc-admin-scanner-trace-desc
+
+The :mc-cmd:`mc admin scanner trace` command displays :ref:`scanner <minio-concepts-scanner>`-specific API operations occurring on the target MinIO deployment.
+
+.. end-mc-admin-scanner-trace-desc
+
+.. admonition:: Use ``mc admin`` on MinIO Deployments Only
+   :class: note
+
+   .. include:: /includes/facts-mc-admin.rst
+      :start-after: start-minio-only
+      :end-before: end-minio-only
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following example returns a list of API operations related to the scanner on the ``myminio`` deployment.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin scanner trace myminio
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin scanner trace ALIAS
+                                [--filter-request]            \
+                                [--filter-response]           \
+                                [--filter-size <value>]       \
+                                [--funcname <value>]          \
+                                [--node <value>]              \
+                                [--path <value>]              \
+                                [--response-duration <value>] \
+                                [--verbose, -v]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :ref:`alias <alias>` of the MinIO deployment for which to display :ref:`scanner <minio-concepts-scanner>` API operations.
+
+.. mc-cmd:: --filter-request
+   :optional:
+
+   Trace client operations or calls with request size greater than the specified :mc-cmd:`~mc admin scanner trace --filter-size` value.
```

Comment:
specific to the scanner right? Maybe

Filter scanner traces to operations or calls with...

Reply:
Yeah, it's scanner-specific ops/calls. I copied this from `admin trace` and missed updating this description.